src/hevm/test/test.hs

```diff
@@ -638,6 +650,7 @@ hex s =
 singleContract :: Text -> Text -> IO (Maybe ByteString)
 singleContract x s =
   solidity x [i|
+    pragma experimental ABIEncoderV2;
```

Comment:
maybe it's worth testing against abiencoderv1 as well?

Reply:
we can't test the interesting types like arrays with abiencoderv1